src/libcoretest/num/flt2dec/mod.rs

```diff
@@ -8,6 +8,10 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// this is in principle private which is only made public for testing. do not
+// expose us in the documentation as a result
+#![doc(hidden)]
```

Comment:
What's the point of this?  Nobody is ever going to generate docs for libcoretest.